iOS Example Frame SPM/iOS Example Frame SPM.xcodeproj/project.pbxproj

```diff
@@ -1211,12 +1219,12 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		16AE74C02A5C1B210031F794 /* XCRemoteSwiftPackageReference "frames-ios" */ = {
+		16900D502AE7508C009A7CE9 /* XCRemoteSwiftPackageReference "frames-ios" */ = {
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/checkout/frames-ios";
 			requirement = {
-				kind = exactVersion;
-				version = 4.2.1;
+				branch = "feature/cvv-component";
+				kind = branch;
```

Comment:
Package pointing to this branch

Reply:
Reason?

Reply:
because the code change is on this branch and the sample apps use the remote reference to fully reflect the developers' experience @deepesh-vasthimal-cko

Reply:
it'll need to be changed back to a version number, with the release PR

Reply:
Got it. thank you. But if you merge this into main then anyone looking at our sample apps will also see CVV work I assume.

Reply:
correct. And I worried about the same and updated the develop branch and now will merge this into develop's updated version to be on the safe side. thanks for calling it out @deepesh-vasthimal-cko

Reply:
I assume this is not a big risk as you are only touching CVV work and will only impact sample app.

Reply:
Yes but still good to be tidy 🙂